commit 28942a0bf1a1a7d0ab59553e1d232ef99b5455ea
Author: Dario Faggioli <dfaggioli@suse.com>
Date:   Tue Nov 10 10:42:59 2020 +0000

    stgbashprompt should not be executable
    
    It is not necessary for contrib/stgbashprompt.sh to have the executable
    permission set.
    
    This was discovered while packaging (for openSUSE) as `rpmlint` reports
    this:
    
      stgit.noarch: W: script-without-shebang /usr/share/stgit/contrib/stgbashprompt.sh
      This text file has executable bits set or is located in a path dedicated for
      executables, but lacks a shebang and cannot thus be executed. If the file is
      meant to be an executable script, add the shebang, otherwise remove the
      executable bits or move the file elsewhere.

diff --git a/contrib/stgbashprompt.sh b/contrib/stgbashprompt.sh
old mode 100755
new mode 100644
